etl-federation/agent-federated.yaml

```diff
@@ -1,3 +1,9 @@
+global:
+  prometheus:
+    enabled: true
+    # fqdn: http://prometheus-operated.monitoring:9090
+  grafana:
+    enabled: false
```

Comment:
Should we also disable Grafana proxy? From my `bad` memory, if the Grafana proxy is not disabled, it could cause some issues.
```
  grafana:
    enabled: false
    proxy: false
```